main.py

```
@@ -19,12 +20,8 @@ def main() -> None:
     Основная функция для запуска бота.
     """
 
-    """
-    Тест корректности смонитрованного тома
-    """
-
-    with open('../videos/volume_up', 'w') as f:
-        f.write(f'Volume is up at {dt.now()}')
+    # Инициализация сервисов
+    user_service = ServiceFactory.get_user_service(USER_DB_CONFIG, ADMIN_CHAT_ID)
```

Ruff (0.8.2)

Local variable `user_service` is assigned to but never used

Remove assignment to unused variable `user_service`

(F841)